xgboost_ray/main.py

```diff
@@ -914,9 +919,12 @@ def handle_actor_failure(actor_id):
     _training_state.training_started_at = time.time()
 
     # Trigger the train function
+    live_actors = [
+        actor for actor in _training_state.actors if actor is not None
+    ]
     training_futures = [
-        actor.train.remote(rabit_args, params, dtrain, evals, *args, **kwargs)
-        for actor in _training_state.actors if actor is not None
+        actor.train.remote(rabit_args, i == 0, params, dtrain, evals, *args,
```

Comment:
could one of the live_actors go away during the training process or fail to return the bst? if so, should we include a retry here to fetch the bst from the next available actor?

Reply:
No, if the designated return actor dies we will restart the `_train()` function in any case and the next live actor will become the new designated return actor